examples/ex1_standalone_app/build.gradle

```diff
@@ -14,5 +14,8 @@ android {
 }
 
 dependencies {
+    // External users should use the repo dependency below instead of the
+    // project dependency.
+    //compile 'com.google.android.mobly:mobly-snippet-lib:1.0.0'
```

Comment:
Yeah, I was referring to public users who are looking at this is example to get started using snippet for their own things. What would be the best way to word that?